src/Stratis.Bitcoin.Features.Interop/EthereumClient/ContractSource/MultisigWallet.sol

```diff
@@ -0,0 +1,392 @@
+pragma solidity ^0.4.15;
+
+
+/// @title Multisignature wallet - Allows multiple parties to agree on transactions before execution.
```

Comment:
Probably add a comment that contract comes from https://github.com/gnosis/MultiSigWallet/blob/master/contracts/MultiSigWallet.sol